src/Prints.cc

```diff
@@ -0,0 +1,92 @@
+#include "Prints.h"
+
+namespace kickcat
+{
+    void printInfo(const Slave& slave)
```

Comment:
To keep the consistency with the convention of the rest of the project prefer using `Slave const&` see what is done in Frame.h or Bus.h.

The code will be working the same, it's just a convention since the rule is "const applies to the thing left of it. If there is nothing on the left then it applies to the thing right of it."